sphinx/builders/latex.py

```diff
@@ -217,7 +216,6 @@ def apply_transforms(self, doctree):
         # type: (nodes.document) -> None
         transformer = SphinxTransformer(doctree)
         transformer.set_environment(self.env)
-        transformer.add_transforms([SubstitutionDefinitionsRemover])
         transformer.apply_transforms()
```

Comment:
I left `SpinxTransformer` as is because I will add another Transform module here later.